client/src/components/ArticlePageMainImageContainer/ArticlePageMainImageContainer.js

```diff
@@ -11,6 +12,7 @@ const Container = styled.div`
 
   img:nth-child(2) {
     width: 100%;
+    height: 13.5rem;
```

Comment:
Look up my previous comment: try to use a parent layout.

Reply:
Done! Thanks!